### PR TITLE
A bit of cleanup

### PR DESCRIPTION
**Pull request by dylanegan**
This is just to clean things up a bit as there were remnants from the Zendesk code that wasn't necessary for this library.

* Remove Zendesk referenced configuration
* Add Standard JS ESLint support
* Adhere to more npm standard packaging and provide distributions for various things

Also the specifications weren't running properly, so I've fixed those to run properly now.

### CODE REVIEW

**Comment by dylanegan** [comment]
@aossowski shall we merge this to master and maybe publish to NPM?

**Comment by aossowski** [comment]
@dylanegan we can merge to master but I think too many resources are still missing there to publish it

**Comment by dylanegan** [comment]
Is that an issue? If it's pre-1.0.0 I wouldn't expect feature parity.

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "test": "./node_modules/karma/bin/karma start --single-run"
   },
   "devDependencies": {
+    "babel-cli": "^6.18.0",
```
Comment (dylanegan) [inline package.json]: What was the requirement here?

File: src/Profile.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Profile {
+  constructor (client) {
+    this.__basePath = '/profile'
+    this.__client = client
+  }
+
+  get () {
+    return this.__client.request(this.__urlFor(''))
```
Comment (dylanegan) [inline src/Profile.js]: Should be able to call it without the argument here.

File: package.json (hunk 1 above)
Comment (aossowski) [inline package.json]: `npm run build` didn't work without it

Comment (dylanegan) [inline package.json]: What was the output without it?

Comment (aossowski) [inline package.json]: `babel: command not found` ;-)

Comment (dylanegan) [inline package.json]: @aossowski no problem, was able to replicate. I must've done `nom install -g babel-cli` as some point on my old machine and my new machine failed immediately. Thanks!

**Comment by dylanegan** [comment]
@aossowski do you foresee any additional endpoints we'll need for Reporting or would this be safe to merge in and move things to using it?

**Comment by aossowski** [comment]
@dylanegan we will need more endpoints to save/retrieve reports and export data but we can add it later as separate PRs

**Pull request merged**